crds/jwst/specs/miri_apcorr.rmap

```diff
@@ -8,8 +8,8 @@ header = {
     'mapping' : 'REFERENCE',
     'name' : 'jwst_miri_apcorr.rmap',
     'observatory' : 'JWST',
-    'parkey' : (('META.EXPOSURE.TYPE',), ('META.OBSERVATION.DATE', 'META.OBSERVATION.TIME')),
-    'sha1sum' : 'e2255db2e45b01b85aa4686d6e1811ad77ff33b8',
+    'parkey' : (('META.EXPOSURE.TYPE', 'META.INSTRUMENT.CHANNEL'), ('META.OBSERVATION.DATE', 'META.OBSERVATION.TIME')),
```

Comment:
I feel like the parkey tuple needs to have it's second setting match, so not sure this is in the right place or if I need a third tuple for just META.INSTRUMENT parkeys.